README.md

```diff
@@ -34,3 +33,17 @@ git checkout main
 git pull
 release-it # Follow the instructions
 ```
+
+### Examples:
+run_ssh_command():
```

Comment:
Please remove this `run_ssh_command():` this is just how to call function in python

Reply:
Ok. Do you want me to remove the line `Enables running a command against a remote server over SSH. It expects a host object, created using [python-rrmngmnt](https://github.com/rhevm-qe-automation/python-rrmngmnt).
` as well? and only keep the sample code?